### PR TITLE
feat: Nonce changes by AA contract

### PR DESCRIPTION
**Pull request by hpsing**
* Nonce by AA user account
* Print Signature value

### CODE REVIEW

File: pkg/ethclient/ethclient.go [hunk 1]
```diff
@@ -7,31 +7,47 @@ import (
...
 // Client wraps the Ethereum client.
 type Client struct {
+	nodeURL   string
 	ethClient *ethclient.Client
 }
 
 // NewClient creates a new Ethereum client.
-func NewClient(nodeURL string) *Client {
-	client, err := ethclient.Dial(nodeURL)
+func NewClient(nodeUrl string) *Client {
```
Comment (blewater) [inline pkg/ethclient/ethclient.go:22]: Go standard for acronyms is all caps :)

**Review by blewater — APPROVED** [review]
LGTM, thx

File: pkg/ethclient/ethclient.go (hunk 1 above)
Comment (singhhp1069) [reply pkg/ethclient/ethclient.go:22]: ops missed it.

**Review by blewater — APPROVED** [review]
LGTM, we may consider #3 in another PR.

**Pull request merged**